packages/api/lambdas/db-indexer.js

```diff
@@ -125,6 +125,8 @@ function getParentId(type, record) {
  * @returns {Promise<Object>} elasticsearch response
  */
 function performIndex(indexFnName, esClient, data) {
+  // TODO: I don't think `process.env.ES_INDEX` is ever set and can be
+  // removed
```

Comment:
Want to do that here or hold off?

Reply:
I'd like to hold off for now since it's not required for my work and I want to avoid unintended consequences